android/assets/jsons/translations/French.properties

```diff
@@ -300,9 +300,9 @@ Lost ability = Compétence perdue
 National ability = Compétence nationale
 [firstValue] vs [secondValue] = [firstValue] vs [secondValue]
  # Requires translation!
-Gained = 
+Gained = Gagne
```

Comment:
It works in french. ^^ "Gagne" _these stats_ or "Perd" _these stats_. (Gain stats / Lose stats)
The active form sounds better in french in this case, rather than the past participle (Gagné / Perdu)

By the way, there's a lot of untranslated stuff in your recent additions (which are great). :D
I'm currently reporting them in the dedicated issue.